packages/commonwealth/client/scripts/helpers/ContractHelpers/ContractBase.ts

```diff
@@ -59,6 +59,13 @@ abstract class ContractBase {
       );
     }
   }
+
+  protected reInitContract() {
```

Comment:
what is the use case for this method?

Reply:
When the contestHelper is instantiated for creating new contests - the address of the new contest does not exist yet so we can create a contract api instance. This method is called/checked before interacting with the contest deployed to instantiate the contract api based on the address generated by `newContest`